src/StatusQ/Controls/StatusPickerButton.qml

```diff
@@ -50,7 +56,7 @@ Button {
             width: parent.width - icon.width - icon.anchors.rightMargin - icon.anchors.leftMargin
             anchors.verticalCenter: parent.verticalCenter
             clip: true
-            spacing: 4
+            spacing: root.iconTextSpacing
```

Comment:
Button already has a `spacing` property, see: https://doc.qt.io/qt-5/qml-qtquick-controls2-control.html#spacing-prop which is meant exactly for this purpose :)